### ISSUE TITLE
I've made it to the dashboard. What now?

### ISSUE THREAD

**Issue by ghost** [mnzt/steam-inviter#2]
What do I do with these functions? 

What am I inviting to?

**Comment by tobbbles** [comment]
Hey, 

This hasn't been touched in a while, my groupID is still hard coded, edit https://github.com/mnzt/steam-inviter/blob/master/app.js#L151 with your group's id (can be found from `https://steamcommunity.com/groups/GROUPNAME/memberslistxml?xml=1` in `<groupID64>`) to get it working.

Currently you have two options - invite from group, and invite from friends list. What these do, is they pull an array of ids from the group's members, or the person's friends list, and sends them invites to your group from the account you logged in to.

**Comment by ghost** [comment]
Ah I see, can I add multiple (100 as a max) groupID64s?

Cheers for the response

**Comment by tobbbles** [comment]
You're wanting to invite people to ~100 groups? Just checking what you mean, otherwise it'd need to be wrote differently to be recursive.

Also, your avatar is familiar, and so are some of the people you follow/favourite'd. Do I know you?

**Comment by ghost** [comment]
> You're wanting to invite people to ~100 groups? 

yeah, or whatever the max invite per day is. So I pick a few groups, and then with your tool, I pick a steam user to invite them to (those 100 groups).

Also, probably not. I lurk around Github/Steam a lot. i;m just a script kiddie tbh

**Comment by tobbbles** [comment]
Hmm, it'd require some extra work on my half. I'll pencil it in to do over the end of the week.

**Comment by ghost** [comment]
Do you know any other alternatives? I don't want to give you more work unless you want to :sweat_smile:

**Comment by tobbbles** [comment]
Currently I don't know of any others. I don't mind, it'd be nice to do some JavaScript work again :smile:, just need to find the time.
